src/APIs/mongo-suspend.ts

```diff
@@ -0,0 +1,13 @@
+import { Document, Schema, model } from 'mongoose';
+
+export interface userSuspend extends Document {
+  userid: string;
+  suspended: boolean;
+}
+
+export const userSchema = new Schema({
+  userid: String,
+  suspended: Boolean
+});
+
+export const userStatus = model<userSuspend>('user', userSchema);
```

Comment:
Change the following variable/property names:
- `userid` to `userId` (for interface and schema)
- `userSuspend` to `UserSuspend`
- `userStatus` to `userModel`

Reply:
Done.

Reply:
Shoot, I missed a few, was distracted when writing this out sorry!
rename
- `userModel` to `userSuspendModel` - in case we keep track of users by themselves in the future
- `userSchema` to `userSuspendSchema` - in case we keep track of users by themselves in the future

Again sorry for not giving recommending better names before :(

Reply:
No worries- all taken care of 🙂